mpact/models.py

```diff
@@ -19,10 +19,44 @@ class Profile(models.Model):
     phone = models.CharField(max_length=20, validators=[phone_regex, validate_phone])
 
 
-class ChatData(models.Model):
-    chat_id = models.CharField(max_length=50)
+class Chat(models.Model):
+    chat_id = models.IntegerField(unique=True)
     title = models.CharField(max_length=350)
     created_at = models.DateTimeField()
 
     def __str__(self):
         return f"{self.chat_id} - {self.title}"
+
+
+class Bot(models.Model):
+    chat = models.OneToOneField(Chat, on_delete=models.CASCADE)
+    bot_id = models.IntegerField()
+    username = models.CharField(max_length=350)
```

Comment:
A 350-character username? That's a lot of typing! It might make sense to find out the max length according to Telegram, or just use a TextField. If we use Postgres as the database, it can still index TEXT columns.

Reply:
yes, it is a lot of typing 😄. I will look into Telegram models and modify it.